frontend/src/app/core/setup/init-locale.ts

```diff
@@ -60,5 +61,7 @@ export function initializeLocale() {
     }
   };
 
+  initializeMomentjsRelativeTimeTranslations(I18n.locale);
```

Comment:
Please do it like this instead:

- Add two files `moment-locale-extensions/en.js` and `moment-locale-extensions/de.js` with contents


```
    const locale = 'en';
    const config = { ... the config file for locale }
    moment.updateLocale(locale, config);
```

- Import the file dynamically with import(/* webpackChunkName: "moment-locale-extension" */ `moment-locale-extensions/I18n.locale.js`) within the timezone service, whenever using the formatRelative() function and return an observable instead.

This ensures this file is actually lazy loaded in case we add multiple languages to it, and will only be loaded when needed.

Reply:
What happens if the config file is not present in the given locale, let's say French? Currently we only offer EN and DE. Then the import will fail, right? Then I want the standard longer version of the French config to stay in place and not get overwritten.

Reply:
That is what should happen as webpack will have a lookup table of available options. I'm not sure if it might result in a console error though, but it's the same approach we currently take for default moment locales + our own locale files